src/script/sign.h

```diff
@@ -67,9 +67,283 @@ struct SignatureData {
     explicit SignatureData(const CScript& script) : scriptSig(script) {}
 };
 
+// Note: These constants are in reverse byte order because serialization uses LSB
+static const uint32_t PSBT_MAGIC_BYTES = 0x74627370;
+static const uint8_t PSBT_UNSIGNED_TX = 0x00;
+static const uint8_t PSBT_REDEEMSCRIPT = 0x01;
+static const uint8_t PSBT_WITNESSSCRIPT = 0x02;
+static const uint8_t PSBT_NON_WITNESS_UTXO = 0x03;
+static const uint8_t PSBT_WITNESS_UTXO = 0x04;
+static const uint8_t PSBT_PARTIAL_SIG = 0x05;
+static const uint8_t PSBT_SEPARATOR = 0x00; // A key that has length 0 is a separator and has no value. That is just 1 byte
```

Comment:
It's a bit confusing that this is a constant like the other ones before, while it's not actually a header byte (which follows a key length of 1 or more), but the length byte itself?